.github/workflows/release_to_stable.yaml

```diff
@@ -0,0 +1,41 @@
+name: Release all FINOS Legend charms to stable
+
+on:
+  workflow_dispatch:
```

Comment:
The idea would be for someone to specify which legend release to promote to stable, which means that that release should be parameterized. Then, that legend release would be promoted from edge to stable (e.g.: ``2022-06/edge`` to ``2022-06/stable``).

Reply:
`yyyy-mm-dd` parameter added.